src/sentry/rules/history/base.py

```diff
@@ -40,7 +42,7 @@ def record(
         group: Group,
         event_id: str | None = None,
         notification_uuid: str | None = None,
-    ) -> None:
+    ) -> Any | None:
```

Comment:
I tried to use `object`, but it did not like the type comparison when made with `RuleFireHistory` model type

Reply:
I think maybe I could use `Type[object]` instead?

Reply:
I think `Any` is fine here. If we wanted to return something more useful, we'd define a dataclass that is separate from the models and return that, I think